mmv1/templates/terraform/custom_flatten/go/bigquery_table_ref.go.tmpl

```diff
@@ -1,4 +1,3 @@
-{{- define "bigqueryTableRegexp"}}
```

Comment:
Sorry, I forgot that I manually converted the templates related to bigquery table. Do you mind cancelling the change in this file?